onnxruntime/python/tools/transformers/fusion_attention.py

```diff
@@ -419,18 +419,27 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
             else:
                 return
 
-        other_inputs = []
-        for i, input in enumerate(start_node.input):
-            if input not in output_name_to_node:
-                continue
```

Comment:
It is better to add a new file fusion_attention_vit.py since its graph pattern seems different. The new class can be derived from this class if needed to reuse some functions.